apps/src/templates/manageStudents/ManageStudentsActionsHeaderCell.jsx

```diff
@@ -0,0 +1,29 @@
+import React, {Component} from 'react';
+import QuickActionsCell from "../tables/QuickActionsCell";
```

Comment:
If you change this line to:
``import QuickActionsCell, {QuickActionsCellType} from "../tables/QuickActionsCell";``
Then on line 12, you can use:
``type={QuickActionsCellType.header}``